src/main/java/randoop/generation/ConstantMiningWrapper.java

```diff
@@ -0,0 +1,140 @@
+package randoop.generation;
+
+import randoop.main.GenInputsAbstract;
+import randoop.sequence.Sequence;
+import randoop.types.ClassOrInterfaceType;
+
+/**
+ * This class is a wrapper for the ConstantMining. The reason why the fields are enumerated for each
```

Comment:
What is "the ConstantMining"?  That is written as if `ConstantMining` is a value being wrapped, but I think you mean "the constant mining algorithm or process".  What exactly is being wrapped?  Is "wrapper" a misnomer?